src/tiles/clickable-tile.component.ts

```diff
@@ -7,6 +7,11 @@ import {
 } from "@angular/core";
 import { Router } from "@angular/router";
 
+/**
+ * Security HTML anchor rel when target is set
+ */
+const REL = "noreferrer noopener";
```

Comment:
We shouldn't have a default like this, it needs to be explicitly passed by caller.